neptune_pytorch_lightning/impl/__init__.py

```diff
@@ -371,21 +370,13 @@ def save_dir(self) -> Optional[str]:
         # Neptune does not save any local files
         return None
 
-    @cached_property
+    @property
     def name(self) -> str:
-        try:
-            self.run.sync()
-        except NeptuneOfflineModeFetchException:
-            return 'offline-name'
-        return self.run['sys/name'].fetch()
+        return 'NeptuneLogger'
 
-    @cached_property
+    @property
     def version(self) -> str:
-        try:
-            self.run.sync()
-        except NeptuneOfflineModeFetchException:
-            return 'offline-id-1234'
-        return self.run['sys/id'].fetch()
+        return repr(__version__)
```

Comment:
`__versions__` value is changing really rare, comparing to `sys/id` generated for each `run`.

Do we have any clue if it's important, do you have example of other loggers which use library version here?

Reply:
I've changed the version property to `_short_id` of a run. When `neptune.init()` is called in offline it's got temporary name for ex. `OFFLINE-1`; In the standard mode, it will fetch for ex. `FAS-123` by itself from Neptune.ai.